### DISCUSSION TITLE
Back Space key customization

### DISCUSSION THREAD

**Discussion by yujustxiaobai** [mulaRahul/keyviz#307] [General]
**Is your feature request related to a problem? Please describe.**
When I got my courses transcribed, I found that I often make mistakes and need to use BackSpace key. Then the display bar will show a lot of backspace keys, which influences the audiences to understand my true content.

**Describe the solution you'd like**
I will be glad if you can add an option that when I use backspace key, the previously displayed letter will be removed and the backspace key will not be displayed at all.

**Comment by mulaRahul** [comment]
I think that's niche, but would love to hear if more people would like the same
